modules/system/lang/vn/lang.php

```diff
@@ -0,0 +1,434 @@
+<?php
+
+return [
+    'app' => [
+        'name' => 'QuangtrongOnlineCMS',
```

Comment:
Please don't add project specific custom translations in the default language files; this is something that should be handled at a project level by making the changes in lang/system/vn/lang.php.